src/config/constants/ifo.ts

```diff
@@ -30,7 +30,9 @@ const ifos: Ifo[] = [
     token: tokens.hzn,
     releaseBlockNumber: 6581111,
     campaignId: '511090000',
-    link: 'https://pancakeswap.medium.com/horizon-protocol-hzn-ifo-to-be-hosted-on-pancakeswap-51f79601c9d8',
+    articleUrl: 'https://pancakeswap.medium.com/horizon-protocol-hzn-ifo-to-be-hosted-on-pancakeswap-51f79601c9d8',
+    tokenOfferingPrice: 0.25,
+    achievementtPoints: 100,
```

Comment:
```suggestion
    achievementPoints: 100,
```

Reply:
@RabbitDoge This can also be retrieved from the contract directly too from `numberPoints`

![image](https://user-images.githubusercontent.com/71833681/114961631-28b97500-9ea4-11eb-9fd1-549accc70f78.png)

Reply:
good catch